kitsune/sumo/static/sumo/js/wiki.js

```diff
@@ -787,4 +787,30 @@
     });
   }
 
+  var $draftButton = $('.btn-draft'),
```

Comment:
Let's take allllll of this javascript you wrote (Lines 790 - 814) and wrap it in a function called `initTranslationDraft` and move it up to line 641 - in between the `initToggleDiff` and `initRevisionList` functions.  Right now it's kind of just hanging out at the bottom of the file, unattached. This will make it a little more consistent with the way the rest of the file is written.

Then let's invoke this new `initTranslationDraft` function on [line 75](https://github.com/safwanrahman/kitsune/blob/draft/kitsune/sumo/static/sumo/js/wiki.js#L75) right beneath `initToggleDiff();`  This will ensure that all of your code is only being used when we're actually on the translation page where it's needed.